<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tienda</class>
 <widget class="QMainWindow" name="Tienda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tienda</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Iconos.qrc">
    <normaloff>:/Recursos2/carro.png</normaloff>:/Recursos2/carro.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 21, 36);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Sans Serif&quot;;</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="inIdiomas">
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>10</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="title">
          <string>Cliente</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;
</string>
            </property>
            <property name="text">
             <string>Cédula:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inCedula">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;
</string>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="inNombre">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Teléfono:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="inTelefono">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>E-mail:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="inEmail">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Dirección:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPlainTextEdit" name="inDireccion">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
            <property name="lineWidth">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="title">
          <string>Producto</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="inProducto">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Cantidad:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="inCantidad">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Sans Serif&quot;;</string>
            </property>
            <property name="text">
             <string>Precio:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="outPrecio">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="btnAgregar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 125, 0);
color: rgb(255, 255, 255);
font: 9pt &quot;Stencil&quot;;</string>
            </property>
            <property name="text">
             <string>Agregar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 97, 109);
font: 87 10pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Detalle de la compra</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="outDetalle">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(234, 234, 236);
font: 10pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnFinalizar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Stencil&quot;;
color: rgb(255, 255, 255);
background-color: rgb(120, 41, 15);</string>
            </property>
            <property name="text">
             <string>Finalizar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="limpiar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Stencil&quot;;
color: rgb(255, 255, 255);
background-color: rgb(120, 41, 15);</string>
            </property>
            <property name="text">
             <string>Nuevo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>187</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="styleSheet">
                <string notr="true">font: 75 9pt &quot;MS Sans Serif&quot;;</string>
               </property>
               <property name="text">
                <string>Subtotal:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="outSubtotal">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 10pt &quot;Consolas&quot;;</string>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="styleSheet">
                <string notr="true">font: 75 9pt &quot;MS Sans Serif&quot;;</string>
               </property>
               <property name="text">
                <string>IVA (12%):</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="outIva">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 75 10pt &quot;Consolas&quot;;</string>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="font">
                <font>
                 <family>Arial Black</family>
                 <pointsize>10</pointsize>
                 <weight>10</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>TOTAL:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="outTotal">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>10</pointsize>
                 <weight>9</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(249, 240, 107);
font: 75 10pt &quot;Consolas&quot;;
color: rgb(224, 27, 36);</string>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Iconos.qrc"/>
 </resources>
 <connections/>
</ui>
